deploy/vmscrape.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: operator.victoriametrics.com/v1beta1
 kind: VMPodScrape
 metadata:
   name: autoscaling-components
-  namespace: monitoring
+  namespace: monitoring-system
```

Comment:
What was the reason for this change? IIUC the metrics scrapes need to be in the `monitoring` namespace in order for the metrics to be routed to the right place.

Reply:
Good catch. I named my local victoria metrics namespace `monitoring-system` and forgot to change it here.